core/tests/karma.conf.js

```diff
@@ -8,7 +8,7 @@ if (isMinificationNeeded) {
 module.exports = function(config) {
   config.set({
     basePath: '../../',
-    frameworks: ['jasmine'],
+    frameworks: ['jasmine-jquery', 'jasmine'],
```

Comment:
Could we add a comment here explaining what specifically jasmine-jquery is needed for, and why jasmine is insufficient?

Reply:
It is used for getting data from a JSON file. I don't know why to use jasmine-jquery and whether the similar functionality is present in jasmine. @anmolshkl might have better answer.

Reply:
@seanlip yes, @prasanna08 is right - jasmine-jquery is used for loading [json](https://github.com/velesin/jasmine-jquery#json-fixtures) and html fixtures in karma tests. I don't know if we'll be able to use getJSONFixture() without including jasmine-jquery.

Reply:
Yep, I'm fine with including it -- I just wanted to ask that the reason be documented (with links if applicable).

Reply:
OK, I will do that in next commit. I am waiting for @anmolshkl's review.